src/wmtk/operations/tri_mesh/EdgeSplitWithTag.cpp

```diff
@@ -44,12 +44,15 @@ Tuple EdgeSplitWithTag::return_tuple() const
 }
 bool EdgeSplitWithTag::execute()
 {
-    // Eigen::Vector3d p0 = m_pos_accessor.const_vector_attribute(input_tuple());
-    // Eigen::Vector3d p1 =
-    // m_pos_accessor.const_vector_attribute(mesh().switch_vertex(input_tuple()));
-    long et = m_edge_tag_accessor.scalar_attribute(input_tuple());
-    long vt0 = m_vertex_tag_accessor.scalar_attribute(input_tuple());
-    long vt1 = m_vertex_tag_accessor.scalar_attribute(mesh().switch_vertex(input_tuple()));
+    // long et = m_edge_tag_accessor.scalar_attribute(input_tuple());
+    long vt0, vt1;
```

Comment:
I really don't like that way of declaring variables and defining them afterwards. Please avoid it if possible and then also make them `const`.